services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationServiceITest.kt

```diff
@@ -68,14 +68,14 @@ import org.springframework.security.test.context.support.WithMockUser
 import org.springframework.test.context.ActiveProfiles
 import org.testcontainers.junit.jupiter.Testcontainers
 
-private const val username = "test7358"
+private const val USER_NAME = "test7358"
```

Comment:
This should probably be USERNAME. I think these should be done in separate pull requests in the future.

Reply:
I changed it to USERNAME as I already touched that.